packages/router/src/config.ts

```diff
@@ -420,7 +420,7 @@ export interface Route {
    */
   component?: Type<any>;
   /**
-   * A URL to which to redirect when a the path matches.
+   * A URL to which to redirect when the path matches.
```

Comment:
```suggestion
   * A URL to redirect to when the path matches.
```
"to which to" appears wrong as well. How about "A URL to redirect to..."?